R/repositories.R

```diff
@@ -249,15 +256,17 @@ BINARY_BASE_URL <- "https://bioconductor.org/packages/%s/container-binaries/%s"
 #'
 #' @md
 #' @export repositories
-repositories <-
-    function(site_repository = character(), version = BiocManager::version())
-{
+repositories <- function(
+    site_repository = character(),
+    version = BiocManager::version(),
+    type = "both"
```

Comment:
I think it would be better for non-required arguments to be after `...` so that the argument must be named and some other 'more important' argument can be inserted before it at some point in the future.

Reply:
Are you saying that the elipses `...` should be added to `repositories()` and that `type="both"` should come after it?

Reply:
yes, to force the use of named arguments.